conn_linux.go

```diff
@@ -47,3 +47,41 @@ func (c *Conn) SetsockoptSockFprog(level, opt int, fprog *unix.SockFprog) error
 
 	return os.NewSyscallError(op, err)
 }
+
+func (c *Conn) GetSockoptTpacketStats(level, name int) (*unix.TpacketStats, error) {
+	const op = "getsockopt"
+
+	var (
+		stats *unix.TpacketStats
+		err   error
+	)
+
+	doErr := c.control(op, func(fd int) error {
+		s, err := unix.GetsockoptTpacketStats(int(fd), level, name)
```

Comment:
Need to avoid `:=` because otherwise err becomes inaccessible to the outside block. Just do `stats, err =`.

Also unnecessary conversion `int(fd)`, can just pass `fd`.